src/redux/actions/users.js

```diff
@@ -1,5 +1,8 @@
-import { ADD_USER, REMOVE_USER } from './types';
+import axios from "axios";
```

Comment:
@Ibidapo Please use the `axiosConfig` instead of `axios` package. Change every occurrence of `axios` in all the `actions` files to `axiosConfig`.